modules/installation-creating-aws-security.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-creating-aws-security-{context}"]
+= Creating security group and roles in AWS
+
+You must create security groups and roles in Amazon Web Services (AWS) for your
+{product-title} cluster to use. The easiest way to create these components is
+to modify the provided CloudFormation template.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS
+infrastructure, you must review the provided information and manually create
+the infrastructure. You might need to consult Red Hat support to ensure that
+your cluster initializes correctly.
+====
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+
+.Procedure
+
+. Create a YAML file that contains a copy of the CloudFormation template that
+describes the networking objects and load balancers and save it to your
+computer.
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
+    "ParameterValue": "mycluster" <2>
+  },
+  {
+    "ParameterKey": "VpcCidr", <3>
+    "ParameterValue": "10.0.0.0/16" <4>
+  },
+  {
+    "ParameterKey": "PrivateSubnets", <5>
+    "ParameterValue": "subnet-0dfe1c9412a54d352" <6>
+  },
+  {
+    "ParameterKey": "VpcId", <7>
+    "ParameterValue": "vpc-055h956b1e6572be9" <8>
+  }
+]
+----
+<1> A short, representative cluster name to use for hostnames, etc.
+<2> Specify the cluster name that you used when you generated the
```

Comment:
cluster name is also used when user decide to generate only ignition-configs files.